File: beginner/__2000__/2813/2813.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int guardachuvaCasa = 0;
        int guardachuvatrabalho = 0;
        int comprasCasa = 0;
        int comprasTrabalho = 0;

        int n = in.nextInt();
        in.nextLine();

        for(int i = 0; i < n; i++){
            String[] input = in.nextLine().split(" ");
            String ida = input[0];
            String volta = input[1];

            if(ida.equals("chuva")){
                if(guardachuvaCasa > 0){ guardachuvaCasa--; } else { comprasCasa++; }
                guardachuvatrabalho++;
            }

            if(volta.equals("chuva")){
                if(guardachuvatrabalho > 0){ guardachuvatrabalho--; } else { comprasTrabalho++; }
                guardachuvaCasa++;
            }
        }

        System.out.println( comprasCasa + " " + comprasTrabalho);
        in.close();
    }
}

File: beginner/__2000__/2813/2813.py
n = int(input())

guardachuvaCasa = 0
guardachuvatrabalho = 0
comprasCasa = 0
comprasTrabalho = 0

for _ in range(n):
    ida, volta = input().split()

    if ida == "chuva":
        if guardachuvaCasa > 0:
            guardachuvaCasa -= 1
        else:
            comprasCasa += 1
        guardachuvatrabalho += 1

    if volta == "chuva":
        if guardachuvatrabalho > 0:
            guardachuvatrabalho -= 1
        else:
            comprasTrabalho += 1
        guardachuvaCasa += 1

print(f"{comprasCasa} {comprasTrabalho}")